{"name": "test_mul[bigint]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 20000 == 30000\n  +20000\n  -30000", "trace": "self = <testing.test_calc_new.Testcalc object at 0x000001A2D0D42A00>, get_calc = <testedcode.calc.Calculator object at 0x000001A2D0CACF10>, get_mul_datas = [100, 200, 30000]\n\n    @allure.story('测试乘法')\n    @pytest.mark.third\n    def test_mul(self,get_calc,get_mul_datas):\n        with allure.step('计算两数相乘'):\n            reslut = get_calc.mul(get_mul_datas[0],get_mul_datas[1])\n>       assert reslut == get_mul_datas[2]\nE       assert 20000 == 30000\nE         +20000\nE         -30000\n\ntest_calc_new.py:46: AssertionError"}, "steps": [{"name": "计算两数相乘", "status": "passed", "start": 1614171803682, "stop": 1614171803682}], "parameters": [{"name": "get_mul_datas", "value": "[100, 200, 30000]"}], "start": 1614171803682, "stop": 1614171803682, "uuid": "e36967a5-3211-4dde-931f-becf068fe1fe", "historyId": "a4e78d7650d7cb93f9bf2ef71431daed", "testCaseId": "00879fb6eeaa4ec170255d63cd763fc1", "fullName": "test_calc_new.Testcalc#test_mul", "labels": [{"name": "story", "value": "测试乘法"}, {"name": "feature", "value": "测试计算器"}, {"name": "tag", "value": "run(order=2)"}, {"name": "tag", "value": "third"}, {"name": "suite", "value": "test_calc_new"}, {"name": "subSuite", "value": "Testcalc"}, {"name": "host", "value": "houss"}, {"name": "thread", "value": "6308-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc_new"}]}